src/libertem/analysis/clust.py

```diff
@@ -24,47 +25,76 @@ class ClusterTemplate(GeneratorHelper):
     def __init__(self, params):
         self.params = params
 
+    def format_code(self, code):
+        code = re.sub(r"self(\.|, |)|(, |)executor|, cancel_id|, job_is_cancelled", "", code)
+        # remove indentation
```

Comment:
There is actually a function available for this in the standard library, as part of the `textwrap` package: [dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) - maybe that can be used instead?